libs/cardano-data/cardano-data.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-data
-version:            1.2.1.0
+version:            1.2.1.1
```

Patch version is for non semantic changes
```suggestion
version:            1.2.2.0
```